### PR TITLE
KOA-5612 Switch to new colours

### PR DESCRIPTION
**Pull request by marianeum**
Remember to include the following changes:

- [x] `UNRELEASED.md` (See [How to write a good changelog entry](https://github.com/Skyscanner/backpack-foundations/blob/main/CHANGELOG_FORMAT.md))
- [x] `README.md`
- [x] Tests
- [x] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

### CODE REVIEW

File: packages/bpk-foundations-common/base/colors/aliases.json [hunk 1]
```diff
@@ -1,73 +1,80 @@
...
-    "SKY_GRAY": "#111236",
-    "BLACK": "#000000",
-    "BLACK_TINT_01": "#1D1B20",
-    "BLACK_TINT_02": "#2C2C2E",
-    "BLACK_TINT_03": "#3A3A3C",
-    "BLACK_TINT_04": "#48484A",
-    "BLACK_TINT_05": "#636366",
-    "BLACK_TINT_06": "#8E8E93",
+    "SKY_BLUE": "#0062E3",
+    "SKY_BLUE_85": "#024DAF",
```
Comment (olliecurtis) [inline packages/bpk-foundations-common/base/colors/aliases.json:5]: What is this token? I don't see it defined anywhere

File: packages/bpk-foundations-common/base/colors/aliases.json [hunk 2]
```diff
@@ -1,73 +1,80 @@
...
+    "BLACK_TINT_01": "#131D2B",
+    "BLACK_TINT_02": "#131D2B",
+    "BLACK_TINT_03": "#243346",
+    "BLACK_TINT_04": "#44505F",
+    "BLACK_TINT_05": "#44505F",
+    "BLACK_TINT_06": "#BDC4CB",
     "PRIMARY_GRADIENT_LIGHT": "#02DDD8",
-    "DARK_SKY": "#02122C",
+    "DARK_SKY": "#05203C",
+    "DARK_SKY_85": "#154679",
```
Comment (olliecurtis) [inline packages/bpk-foundations-common/base/colors/aliases.json:44]: Same here

File: packages/bpk-foundations-common/base/colors/aliases.json [hunk 3]
```diff
@@ -1,73 +1,80 @@
...
-    "NIGHT_GREY_10": "#000000",
-    "NIGHT_GREY_20": "#1D1B20",
-    "NIGHT_GREY_25": "#2C2C2E",
-    "NIGHT_GREY_30": "#48484A",
-    "NIGHT_GREY_40": "#8E8E93",
-    "WHITE_ALPHA_20": "#8E8E93",
-    "WHITE_ALPHA_80": "#8E8E93",
-    "BLACK_ALPHA_20": "#B2B2BF",
-    "BLACK_ALPHA_70": "#02122c"
+    "NIGHT_GREY_00": "#0B121D",
```
Comment (olliecurtis) [inline packages/bpk-foundations-common/base/colors/aliases.json:68]: Can we check with Adam on this one as I know he removed some aliases recently and this was one as we didn't need some of the additional ones

**Review by olliecurtis — CHANGES_REQUESTED** [review]

File: packages/bpk-foundations-common/base/colors/skeleton.json [hunk 4]
```diff
@@ -15,7 +15,7 @@
       "value": "#FFFFFF99"
     },
     "PRIVATE_SKELETON_SHIMMER_CENTER_NIGHT": {
-      "value": "#00000033"
+      "value": "{!BLACK_ALPHA_20}"
```
[the remark this answers is not in the document]
Comment (marianeum) [reply packages/bpk-foundations-common/base/colors/skeleton.json]: sticking to not using aliases for these as per design

**Review by olliecurtis — APPROVED** [review]

**Pull request merged**